WebApp-B2C-DotNet/Web.config

```diff
@@ -9,13 +9,13 @@
     <add key="webpages:Enabled" value="false" />
     <add key="ClientValidationEnabled" value="true" />
     <add key="UnobtrusiveJavaScriptEnabled" value="true" />
-    <add key="ida:Tenant" value="[Enter the name of your B2C directory, e.g. contoso.onmicrosoft.com]" />
-    <add key="ida:ClientId" value="[Enter the Application Id assinged to your app by the Azure portal, e.g.580e250c-8f26-49d0-bee8-1c078add1609]" />
-    <add key="ida:AadInstance" value="https://login.microsoftonline.com/{0}{1}{2}" />
+    <add key="ida:Tenant" value="fabrikamb2c.onmicrosoft.com" />
+    <add key="ida:ClientId" value="90c0fe63-bcf2-44d5-8fb7-b8bbc0b29dc6" />
```

Comment:
It's not.  This is a sample tenant that developers can use if they want to just download the completed code and immediately run it.  Our plan is to move these "quick starts" back to github.com/Azure-Samples, and for the READMEs there to have dead simple instructions on how to get the app running, instead of a long tutorial about how it works.